nodeletctl/pkg/nodeletctl/templates.go

```diff
@@ -36,9 +36,9 @@ DOCKERHUB_ID: ""
 DOCKERHUB_PASSWORD: ""
 ENABLE_CAS: "false"
 ENABLE_PROFILE_AGENT: "true"
-ETCD_DATA_DIR: /var/opt/pf9/kube/etcd/data
-ETCD_DISCOVERY_URL: ""
-ETCD_ELECTION_TIMEOUT: "1000"
+ETCD_DATA_DIR: {{ or .EtcdDataDir "/var/opt/pf9/kube/etcd/data" }}
```

Comment:
I am wondering if its better to add these defaults in the code (NodeletConfig sructure) rather than the templating language? Simlar to how I set some Calico defaults.

Reply:
Moved it over.